### ISSUE TITLE
Inguinal abscess

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6652]
**Preferred term label:**
Inguinal abscess

**Synonyms**
Groin abscess

**Definition (free text, please give PubMed ID)**
Abscesses involving the inguinal region.
PMID: 23537455

**Parent term (use hpo.jax.org/app)**
Abscess

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 610738

**Comment by pnrobinson** [comment]
added term, thanks!
